tests/functional/common.bash.in

```diff
@@ -34,7 +34,7 @@ CONFIG="${TEST_DATA_DIR}/config-minimal-cc-oci.json"
 HYPERVISOR_ARGS="${BATS_TEST_DIRNAME}/../../data/hypervisor.args"
 
 BUNDLE_DIR="@BUNDLE_TEST_PATH@"
-ROOTFS_DIR="@BUNDLE_TEST_PATH@"
+[ -n "$BUNDLE_DIR" ] && ROOTFS_DIR="$BUNDLE_DIR/rootfs"
```

Comment:
In that case, `ROOTFS_DIR` would also be empty. That's what we want as `ceck_ccontainers()` checks both variables and if they don't exist as directories (they won't if they are unset), the tests that needs bundles/rootfs's are skipped.